plenary/plenary-10.yaml

```diff
@@ -0,0 +1,294 @@
+metadata:
+  title: Resolutions adopted at the meeting of ISO/TC 154 in Cologne
+  date: 
+    start: 1995-08-31
+    end: 1995-09-01
+  source: ISO/TC 154 Secretariat
+resolutions:
+  - dates: 
+      - 1995-08-31
+      - 1995-09-01
+    subject: ISO/TC 154 "Documents and Data Elements in Administration, Commerce and Industry"
+    identifier: 88
+    
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: UNANIMOUS
+        
+    actions:
+      - type: scopes
+        date_effective:
+          start: 1995-08-31
+          end: 1995-09-01
+        message: |
+        The current scope of ISO/TC is:
+        "Standardization of documents and representation of data used for information interchange within administration, commerce and industry. Excluded are documents intended to be read by machines only, fallin into the responsibility of ISO/IEC JTC 1."
+        
+        # TC 154 wants to give the following interpretation of its present scope:
+        
+        # Bodies other than ISO/IEC JTC 1 are responsible for standards in the area of machine readable data and shall be added to the exclusion.
+        
+        # Also excluded from the scope are:
+        
+        # 1 Development of methodology and registration procedures
+        # 2 Industrial data (related to manufacturing or production industrial activities)
+        
+        # Within the scope is work based on the following and which, where relevant, will be developed into international standards following, when available, existing standardized methodologies:
+        
+        # 1 Provide and/or offer coordination for application oriented data.
+        # 2 Specify requirements to improve the use of EDI in business processes in administration, commerce and industry. Based on these requirements, register and where appropriate develop into International Standard EDI scenarios and data structures (e.g. messages, segments and data from UN/EDIFACT TDID) for administration, commerce and industry.
+        # 3 For application oriented EDI scenarios and data structures, developed outside of administration, commerce and industry by other ISO/TCs or external bodies, provide coherency (e.g. between the use of data on paper and in EDI) and assist in resolving issues between technical and business data
+        # 4 Maintain existing ISO/TC 154 standards such as the EDIFACT syntax (ISO 9735) AND TDED (ISO 7372) in accordance with the Memorandum of Understanding (MoU) between IEC, ISO and UN/ECE concerning standardization in the field of EDI and Open-edi.
```

Comment:
I'm not sure how to set this resolution properly. I have commented most of the text. Could you please provide me some feedback in this case?

Original source:
![capture15](https://user-images.githubusercontent.com/33627611/71760068-e6764b00-2e8d-11ea-963b-a6705289b0ec.JPG)

Reply:
We can just set them as AsciiDoc:

```yaml
        message: |
          The current scope of ISO/TC is:
          _"Standardization of documents and representation of data used for
          information interchange within administration, commerce and
          industry_.

          _Excluded are documents intended to be read by machines
          only, falling into the responsibility of ISO/IEC JTC 1."_

          TC 154 wants to give the following interpretation of its present
          scope:

          Bodies other than ISO/IEC JTC 1 are responsible for standards in
          the area of machine readable data and shall be added to the
          exclusion.

          Also excluded from the scope are:

          1. Development of methodology and registration procedures

          2. Industrial data (related to manufacturing or production
          industrial activities)

          Within the scope is work based on the following and which, where
          relevant, will be developed into international standards following,
          when available, existing standardized methodologies:

          1. Provide and/or offer coordination for application oriented data.

          2. Specify requirements to improve the use of EDI in business
          processes in administration, commerce and industry. Based on these
          requirements, register and where appropriate develop into
          International Standard EDI scenarios and data structures (e.g.
          messages, segments and data from UN/EDIFACT TDID) for
          administration, commerce and industry.

          3. For application oriented EDI scenarios and data structures,
          developed outside of administration, commerce and industry by other
          ISO/TCs or external bodies, provide coherency (e.g. between the use
          of data on paper and in EDI) and assist in resolving issues between
          technical and business data

          4. Maintain existing ISO/TC 154 standards such as the EDIFACT
          syntax (ISO 9735) AND TDED (ISO 7372) in accordance with the
          Memorandum of Understanding (MoU) between IEC, ISO and UN/ECE
          concerning standardization in the field of EDI and Open-edi.
````